services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/events.py

```diff
@@ -1,5 +1,235 @@
+        network_config = {
+            "Name": scheduler_data.dynamic_sidecar_network_name,
+            "Driver": "overlay",
+            "Labels": {
+                "io.simcore.zone": f"{dynamic_sidecar_settings.TRAEFIK_SIMCORE_ZONE}",
+                "com.simcore.description": f"interactive for node: {scheduler_data.node_uuid}",
+                "uuid": f"{scheduler_data.node_uuid}",  # needed for removal when project is closed
+            },
+            "Attachable": True,
+            "Internal": False,
+        }
+        dynamic_sidecar_network_id = await create_network(network_config)
+
+        # attach the service to the swarm network dedicated to services
+        swarm_network: Dict[str, Any] = await get_swarm_network(
+            dynamic_sidecar_settings
+        )
+        swarm_network_id: str = swarm_network["Id"]
+        swarm_network_name: str = swarm_network["Name"]
+
+        # start dynamic-sidecar and run the proxy on the same node
+        dynamic_sidecar_create_service_params = await get_dynamic_sidecar_spec(
+            scheduler_data=scheduler_data,
+            dynamic_sidecar_settings=dynamic_sidecar_settings,
+            dynamic_sidecar_network_id=dynamic_sidecar_network_id,
+            swarm_network_id=swarm_network_id,
+            settings=settings,
+        )
+        logger.debug(
+            "dynamic-sidecar create_service_params %s",
+            pformat(dynamic_sidecar_create_service_params),
+        )
+
+        dynamic_sidecar_id = await create_service_and_get_id(
+            dynamic_sidecar_create_service_params
+        )
+
+        dynamic_sidecar_node_id = await get_node_id_from_task_for_service(
+            dynamic_sidecar_id, dynamic_sidecar_settings
+        )
+
+        dynamic_sidecar_proxy_create_service_params = await get_dynamic_proxy_spec(
+            scheduler_data=scheduler_data,
+            dynamic_sidecar_settings=dynamic_sidecar_settings,
+            dynamic_sidecar_network_id=dynamic_sidecar_network_id,
+            swarm_network_id=swarm_network_id,
+            swarm_network_name=swarm_network_name,
+            dynamic_sidecar_node_id=dynamic_sidecar_node_id,
+        )
+        logger.debug(
+            "dynamic-sidecar-proxy create_service_params %s",
+            pformat(dynamic_sidecar_proxy_create_service_params),
+        )
+
+        # no need for the id any longer
+        await create_service_and_get_id(dynamic_sidecar_proxy_create_service_params)
+
+        # update service_port and assing it to the status
+        # needed by CreateUserServices action
+        scheduler_data.service_port = extract_service_port_from_compose_start_spec(
+            dynamic_sidecar_create_service_params
+        )
+
+        # finally mark services created
+        scheduler_data.dynamic_sidecar.were_services_created = True
+
+
+class GetStatus(DynamicSchedulerEvent):
+    """
+    Inspects the dynamic-sidecar, and store some information about the contaiers.
```

Comment:
you might want to adjust this comment."store some information about the containers" is a bit vague... which containers? the sidecar ones or the whole stack? it basically return the status, composed of the sidecars and the attached services right?

Reply:
updated docstring